[release-danube] style spec (#9706)
mapbox/mapbox-gl-js: src/style-spec/CHANGELOG.md

## 13.15.0

### ✨ Features and improvements

> * Add `distance` expression entry in `style-spec`, which returns the shortest distance between a feature and an input geometry ([#9655](https://github.com/mapbox/mapbox-gl-js/pull/9655))

Reviewer: ```suggestion
* Add `distance` expression to `style-spec`. This expression returns the shortest distance between a feature and an input geometry ([#9655](https://github.com/mapbox/mapbox-gl-js/pull/9655))
```
